src/test/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTriggerTest.java

```diff
@@ -892,6 +898,120 @@ public void testGerritEventManualEvent() {
     }
 
     /**
+     * Tests {@link EventListener#gerritEvent(com.sonymobile.tools.gerrit.gerritevents.dto.GerritEvent)}.
+     * With a RefUpdated event with short ref name.
+     */
+    @Test
+    public void testGerritEventRefUpdatedShortName() {
+        AbstractProject project = mockProject();
+
+        Queue queue = mockConfig(project);
+
+        PowerMockito.mockStatic(ToGerritRunListener.class);
+        ToGerritRunListener listener = PowerMockito.mock(ToGerritRunListener.class);
+        PowerMockito.when(ToGerritRunListener.getInstance()).thenReturn(listener);
+
+        List<Branch> branches = new ArrayList<Branch>();
+        Branch br = new Branch();
+        br.setCompareType(CompareType.PLAIN);
+        br.setPattern("master");
+        branches.add(br);
+
+        GerritProject gP = new GerritProject(
+                CompareType.PLAIN, "job", branches, null, null, null);
+
+        GerritTrigger trigger = Setup.createRefUpdatedTrigger(project);
+        Setup.setTrigger(trigger, project);
+        trigger.setGerritProjects(Collections.nCopies(1, gP));
+        trigger.setEscapeQuotes(false);
+        trigger.setSilentMode(false);
+        Whitebox.setInternalState(trigger, "job", project);
```

Comment:
this line is in essence ```trigger.job = project;``` so I don't think that has anything to do with the plain matcher?